web-api/src/persistence/postgres/utils/migrate/migrations/2025-06-24T17_12_01Z-user-table.ts

```diff
@@ -0,0 +1,126 @@
+    .addColumn('practitionerNotes', 'varchar')
+    .addColumn('practitionerType', 'varchar')
+    .addColumn('suffix', 'varchar')
+    .addColumn('updatedEmail', 'varchar')
+    .addColumn('accountStatus', 'varchar', col => col.notNull())
+    .execute();
+
+  await db.schema
+    .createIndex('idx_user_section')
+    .on('dwUser')
+    .column('section')
+    .execute();
+
+  await db.schema
+    .createIndex('idx_user_bar_number')
+    .on('dwUser')
+    .column('barNumber')
+    .execute();
+
+  await db.schema
+    .createIndex('idx_user_role')
+    .on('dwUser')
+    .column('role')
+    .execute();
+
+  await db.schema
+    .createIndex('idx_user_name')
+    .on('dwUser')
+    .column('name')
+    .execute();
+
+  // dwUserOnCasePending Table
+  await db.schema
+    .createTable('dwUserOnCasePending')
+    .addColumn('userId', 'varchar', col => col.notNull())
+    .addColumn('docketNumber', 'varchar', col => col.notNull())
+    .addPrimaryKeyConstraint('pkUserOnCasePending', ['docketNumber', 'userId'])
+    .execute();
+
+  // dwUserOnCase Table
+  await db.schema
+    .createTable('dwUserOnCase')
+    .addColumn('userId', 'varchar')
+    .addColumn('docketNumber', 'varchar')
+    .addColumn('representing', 'jsonb')
+    .addColumn('serviceIndicator', 'varchar')
+    .addColumn('actingAsRole', 'varchar', col => col.notNull())
+    .addPrimaryKeyConstraint('pk_user_on_case', ['userId', 'docketNumber'])
+    .execute();
+
+  await db.schema
+    .createIndex('idx_user_on_case_docket_number')
+    .on('dwUserOnCase')
+    .column('docketNumber')
+    .execute();
+
+  // dwUserConfirmationCode Table
+  await db.schema
+    .createTable('dwUserConfirmationCode')
+    .addColumn('userId', 'varchar', col => col.primaryKey())
+    .addColumn('confirmationCode', 'varchar', col => col.notNull())
+    .addColumn('ttl', 'bigint', col => col.notNull())
+    .execute();
+
+  await db.schema
+    .createIndex('idx_confirmation_code_ttl')
+    .on('dwUserConfirmationCode')
+    .column('ttl')
+    .execute();
+
+  // dwBarNumber
+  await db.schema
+    .createTable('dwBarNumber')
+    .addColumn('year', 'int8', col => col.primaryKey())
+    .addColumn('lastUsedNumber', 'int8', col => col.notNull())
+    .execute();
+}
+
+export async function down(db: Kysely<any>): Promise<void> {
+  // Drop indexes in reverse order of creation
+  await db.schema.dropIndex('idx_confirmation_code_ttl').execute();
+  await db.schema.dropIndex('idx_user_on_case_docket_number').execute();
+  await db.schema.dropIndex('idx_user_name').execute();
+  await db.schema.dropIndex('idx_user_role').execute();
+  await db.schema.dropIndex('idx_user_bar_number').execute();
+  await db.schema.dropIndex('idx_user_section').execute();
+
+  // Drop tables
+  await db.schema.dropTable('dwUserOnCasePending').execute();
+  await db.schema.dropTable('dwUserConfirmationCode').execute();
+  await db.schema.dropTable('dwUserOnCase').execute();
+  await db.schema.dropTable('dwUser').execute();
```

Comment:
Ok...so we're not gonna have to roll this back, because we Always Fail Forward. ™️ 

But let's say IRL we really actually need to run this `down` due to catastrophic unforeseen circumstances. That's truly all it's here for, right? So in that incredible unlikely event, shouldn't we rename these tables so we don't lose the data?